flowtools/flowai/FCSflowAI.xml

```diff
@@ -1,13 +1,14 @@
-<tool id="flowAI" name="flowAI" version="1.0">
+<tool id="flowAI" name="flowAI" version="1.0+galaxy1">
   <description> automatic quality control</description>
   <requirements>
-    <requirement type="package" version="2.0.2">ig-flowtools</requirement>
+    <requirement type="package" version="1.42.0">bioconductor-flowcore</requirement>
```

Comment:
My only concern is whether we should stick to a specific version of flowcore or go with whatever flowcore version flowai wants to use. Do you know anyone in the field that normally uses these packages that could advice @emily101-gif ? My question is whether it might be relevant to pin flowcore across all modules to the same version or not.

Reply:
Hi Pablo - I'm afraid I don't know any experts other than just contacting bioconductor. But for many of the tools when I tried to use alternative versions of flowCore or other packages failed. So does it seem safer to use the package version immport use which work and are also working on planemo testing?

Reply:
which version did you try? What I mean here is not to change the version of flowcore manually, but to just use flowai dependency on its own and let conda decide which version of flowcore to install (since flowai depends on flowcore in conda).

Reply:
Based on the comment of the maintainers from flowcore, I'm going to stick to a single dependency. Will merge like this is tests pass.